domain/consensus/processes/transactionvalidator/transactionvalidator_test.go

```diff
@@ -79,8 +80,18 @@ func TestValidateTransactionInContextAndPopulateMassAndFee(t *testing.T) {
 				true,
 				uint64(5)),
 		}
-		txInputWithMaxSequence := externalapi.DomainTransactionInput{
-			PreviousOutpoint: prevOutPoint,
+		txInputWithSecondsSequence1 := externalapi.DomainTransactionInput{
+			PreviousOutpoint: prevOutPoint1,
+			SignatureScript:  []byte{},
+			Sequence:         constants.SequenceLockTimeIsSeconds,
+			UTXOEntry: utxo.NewUTXOEntry(
+				100000000, // 1 KAS
```

Comment:
Please add underscores here and in line 98 to make the number a bit more readable.